### DISCUSSION TITLE
Apicurio registry support

### DISCUSSION THREAD

**Discussion by smeana** [provectus/kafka-ui#1177] [Q&A] [answered]
Hello 

We are trying to have the UI working with Apicurio but the UI cant deserialize the avro messages.
Any advise on how to integrate Apicurio?

regards

**Comment by Haarolean** [comment]
Hi, tanks for the interest in kafka-ui. 
Have you tried using schema registry at all?

**Reply by smeana** [reply]
Hello 

Yes, with the schema registry it is working fine but with Apicurio the Ui cannot access the schemas.

Regards

**Reply by Haarolean** [reply]
Okay, I'll convert this discussion into an issue since we've got a similar issue with some other provider. Thanks!

**Comment by Haarolean** [comment, marked as answer]
The app works with apicurio [just fine](https://github.com/provectus/kafka-ui/issues/1621).
